docs/89_glossary.md

```diff
@@ -126,9 +126,69 @@ following requirements:
   * Something you know (Example: passwords)
   * Something you are (Example: biometrics)
 
-* **wrong outcome** – When the reported outcome does not match the
-actual outcome. This can happen due to various reasons like
-adjudication or equipment errors.
+* **wrong outcome** - When the reported outcome does not match the
+outcome that a full hand count would show.
+This can happen due to equipment failures, adjudication errors,
+and other reasons.
+
+* **evidence-based elections* - an approach to achieving election integrity
```

Comment:
Nit: As above, inconsistent sentence capitalization for the start of the definition.